sky/packages/sky/lib/src/rendering/flex.dart

```diff
@@ -21,29 +28,61 @@ class FlexParentData extends BoxParentData with ContainerParentDataMixin<RenderB
   String toString() => '${super.toString()}; flex=$flex';
 }
 
-enum FlexDirection { horizontal, vertical }
+/// The direction in which the box should flex
+enum FlexDirection {
+  /// Children are arranged horizontally, from left to right
```

Comment:
we really should have horizontal-ltr and horizontal-rtl, but i guess that's a patch for another day